firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/schema.ts

```diff
@@ -148,4 +148,4 @@ export const RawChangelogSchema: any = {
     },
     documentIdField,
   ],
-};
+};
```

Comment:
line ending artifact here - running `npm format` at the repository root might take care of it